crates/solidity/inputs/language/bindings/rules.msgb

```diff
@@ -49,6 +49,12 @@ inherit .star_extension
     ; special symbol (which is automatically imported below)
     attr (export) pop_symbol = "@@built-ins@@"
 
+    ; NOTE: since we're no longer ingesting built-ins from an external Solidity
```

Comment:
> how to link to a fixed, built-in scope

what would be a use case for that?

If none, since both the public API and the bindings internal code are now using "built-ins", should we rename "system files" to "built-in files" to avoid confusion?

Reply:
That's a good point. I don't think we'll need `add_system_file` that in the public API either. I can't think of any reason why we would need it. And then we can remove the code in the rules and graph builder functions to deal with that as well.